modules/proc_configuring-annotation-based-tracking-in-multiple-argo-cd-instances.adoc

```diff
@@ -0,0 +1,149 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-annotation-based-tracking-in-multiple-argo-cd-instances_{context}"]
+= Configuring annotation-based tracking in multiple Argo CD instances
+
+You can configure annotation-based tracking in multiple Argo CD instances.
```

Comment:
Since this proc uses example values instead of user-replaced values, it might be good to call out what the examples will be up front (to avoid users just copying these verbatim into their UI)

Reply:
Could you elaborate and provide an example for this comment?